specification/resources/databases/get_database_config.yml

```
@@ -0,0 +1,41 @@
+operationId: get_database_config
+
+summary: Retrieve Configuration for an Existing Database
+
+description: |
+  To show configuration parameters for an existing database cluster, send a GET request to
+  `/v2/databases/$DATABASE_ID/config`.
+
+  Note: Database config is not yet supported for Redis and Postgres clusters.
```

```suggestion
  **Note:** Database config is not yet supported for Redis and Postgres clusters.
```

It looks like the word `NOTE:` is bolded in the rest of the Database files.